pallets/messages/src/types.rs

```diff
@@ -62,6 +62,15 @@ where
 					msa_id: None,
 					payload: None,
 				}
+			}, // TODO: Add support for itemized and paginated payloads
```

Comment:
One question is that do we really want to expose these stateful storages as messages. I'm not totally convinced about that since we have a different definition of a message and stateful storage and even the access is from different pallets. maybe we want to have a more generic return type that will include all types but then that should have a different name than `MessageResponse`

Reply:
Hmm. Well I think the thing is, we're not storing *messages* in stateful storage; we're storing a mutable final state. The *message* will (may) be generated when stateful storage is modified, but the `message payload <> stateful storage payload`. Put simply, I think we should generate an error if we try to store a `Message` in stateful storage. The `Message` would be something like `StatefulStorageUpdated` (or, more likely something more specific like `PrivateGraphUpdated`, and *that* message's storage location would be `OnChain`.

Reply:
this is just an RPC call to get messages , I have added a comment that for Itemized and Paginated its a different access point but returning default MessageResponse so we dont have to update messages pallets 
What you think

Reply:
I guess the question is, should we return a `MessageResponse` with `None` values, or should we return an error, since there should be no "messages" in Itemized or Paginated storage?

For instance, maybe `get_message_by_schema_and_block()` should just return an empty vector if the storage location specified in the call is Itemized or Paginated?

Reply:
aah I see what you mean, I will add those checks and bypass